app/Http/Resources/QuestionPromo.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+use Illuminate\Support\Str;
+
+class QuestionPromo extends JsonResource
```

Comment:
What does the promo mean there?
This is a confusing name for the resource. Please rename it to something else, maybe QuestionShort, QuestionReduced, etc.